src/Security/Authentication/Twitter/src/TwitterHandler.cs

```diff
@@ -41,12 +44,18 @@ protected new TwitterEvents Events
             set { base.Events = value; }
         }
 
+        /// <summary>
+        /// Initializes a new instance of <see cref="TwitterHandler"/>.
+        /// </summary>
+        /// <inheritdoc />
```

Comment:
```suggestion
```

Reply:
My understanding is that this copies the param docs, I copied this from the facebook handler ctor